.github/workflows/build.yml

```diff
@@ -16,21 +16,7 @@ on:
       - 'bin/**'
       - 'nix/**'
       - 'default.nix'
-  pull_request:
```

Comment:
Notably this actives "on" the merge of the PR branch and its target, if I understand the prior behavior and [the docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) correctly. I would expect it to still be desirable to automate running tests against the would-be end result of a merge, so unclear to me why this got removed.

Reply:
Ah, i didn't realize that.  Will re-add